### PR TITLE
Feature/1878/cytoscape popper

### PR DESCRIPTION
**Pull request by garyluu**
For ga4gh/dockstore#1878

- Fix the import of cytoscape and cytoscape-dagre to be less hacky.
- Add cytoscape type definitions
- Remove cytoscape qtip in favor of cytoscape-popper
- Made the tooltip slightly translucent https://github.com/ga4gh/dockstore/issues/762
- Change the z-index of the DAG full screen to match what a bootstrap modal has, the z-index of the tooltip matches the z-index of a bootstrap tooltip https://getbootstrap.com/docs/4.1/layout/overview/#z-index
- Update some test deps because for some reason it didn't like the less hacky import...?

### CODE REVIEW

**Comment by garyluu** [comment]
codacy: nothing can be done about that
codecov: I didn't even change those files

File: src/app/workflow/dag/dag.component.scss [hunk 1]
```diff
@@ -14,3 +14,4 @@
  *    limitations under the License.
  */
 
+ @import url(https://cdnjs.cloudflare.com/ajax/libs/qtip2/3.0.3/jquery.qtip.min.css);
```
Comment (agduncan94) [inline src/app/workflow/dag/dag.component.scss]: :cloud: :fire:

File: src/app/workflow/dag/dag.component.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export class DagComponent extends EntryTab implements OnInit, OnChanges {
   @ViewChild('cy') cyElement: ElementRef;
 
   public dagResult$: Observable<any>;
-  private cy: any;
+  private cy: cytoscape.Core;
```
Comment (coverbeck) [inline src/app/workflow/dag/dag.component.ts]: 👍

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "start": "ng serve",
     "prebuild:tsnode": "ts-node git.version.ts",
     "prebuild:downloadcodegen": "wget --no-verbose http://central.maven.org/maven2/org/openapitools/openapi-generator-cli/3.3.1/openapi-generator-cli-3.3.1.jar -O swagger-codegen-cli.jar",
-    "prebuild:generatecode": "java -jar swagger-codegen-cli.jar generate -i https://raw.githubusercontent.com/ga4gh/dockstore/${WEBSERVICE_VERSION}/dockstore-webservice/src/main/resources/swagger.yaml -l typescript-angular -o src/app/shared/swagger -c swagger-config.json",
```
Comment (denis-yuen) [inline package.json]: FYI, ran into an issue following the readme. 
This variable doesn't exist outside of Travis-CI.

**Pull request merged**